position.lua

```diff
@@ -55,7 +55,30 @@ end
 print('Place:')
 print('    The z-level is z='..df.global.window_z)
 print('    The cursor is at x='..cursor.x..', y='..cursor.y)
-print('    The window is '..df.global.gps.dimx..' tiles wide and '..df.global.gps.dimy..' tiles high')
-if df.global.gps.mouse_x == -1 then print('    The mouse is not in the DF window') else
-print('    The mouse is at x='..df.global.gps.mouse_x..', y='..df.global.gps.mouse_y..' within the window') end
---TODO:  print('    The fortress is at '..x, y..' on the world map ('..worldsize..' square)')
+print('    The window is '..df.global.gps.dimx..' tiles wide and '..df.global.gps.dimy..' tiles high.')
+
+if df.global.gps.mouse_x < 0 then
```

Comment:
Ended up putting this on a separate line. Let me know if it looks okay. It says "The mouse is" twice in a row, but it's better than doing a comma+indent, IMO.

Reply:
Could also add an option to copy mouse tile position to clipboard. However there's limited feedback of where the mouse cursor is on the grid, and it would be difficult to use with the full launcher (which blocks most of the screen).